build.sbt

```diff
@@ -57,6 +57,15 @@ addCommandAlias(
   ).mkString(";", ";", "")
 )
 
+TaskKey[Unit]("check-clean") := {
```

Comment:
Can we use `checkClean`? This syntax is deprecated.

Reply:
We agreed to remove that task altogether.